### ISSUE TITLE
Can't be compiled using esbuild

### ISSUE THREAD

**Issue by zachlatta** [hackclub/gamelab#88]
Today I was trying to help a Hack Clubber who has a lot of websites blocked by their school on their computer. They are unable to load gamelab.hackclub.com, and other domains are blocked too. For example, https://cdn.skypack.dev/md5 - which is a dynamic module linked from our code - is blocked by their school.

I was thinking I could use esbuild to create a JavaScript bundle to make a 100% local build of Game Lab and load it onto their computer using a USB stick. But I can't build Game Lab using esbuild because it has dynamic external modules, which esbuild doesn't support.

**Comment by cjdenio** [comment]
@sporeball Game Lab uses fancy browser-native ESM, which isn't `package.json`-aware. ejs and axios are just dependencies used by Vercel serverless functions

I think the biggest thing preventing Game Lab from being bundled is that Muse isn't on npm: https://github.com/hackclub/gamelab/blob/19901deb841362821dd324bb16b2ab8a31a47321/assets/ui-sounds.js#L1

**Comment by cedric-h** [comment]
While the way we import our dependencies is not compatible with esbuild, it is more in line with how deno does things. I suspect you might have better luck with `deno --bundle`. If not, please open another issue and we'll get on it.

**Comment by zachlatta** [comment]
Awesome! Can that be tested and added to the README?
